components/tabs/Cargo.toml

```diff
@@ -21,8 +21,7 @@ serde_derive = "1"
 serde_json = "1"
 sql-support = { path = "../support/sql" }
 sync-guid = { path = "../support/guid", features = ["random"] }
-sync15 = { path = "../sync15", optional = true }
-sync15-traits = { path = "../support/sync15-traits" }
+sync15 = { path = "../sync15", features = ["sync-engine"] }
```

Comment:
should this be `standalone-sync` or is the flag on line 10 enough for us to realize later?